data/datavalid.yml

```diff
@@ -402,6 +402,15 @@ schemas:
       - name: appealed
       - name: agency
       - name: motions
+      - name: appeal_disposition_year
+      - name: appeal_disposition_month
+      - name: appeal_disposition_day 
+      - name: appeal_hearing_year
+      - name: appeal_hearing_month
+      - name: appeal_hearing_day
+      - name: appeal_hearing_2_year 
+      - name: appeal_hearing_2_month
+      - name: appeal_hearing_2_day
```

Comment:
Can we replace these columns with "appeal_disposition", "appeal_hearing_1", "appeal_hearing_2" events instead?

Reply:
Replaced.